pkg/pillar/scripts/generate-self-signed.sh

```diff
@@ -91,7 +94,9 @@ esac
 
 if [ "$use_tpm" = true ]; then
     echo "Writing device certificate to TPM"
-    /opt/zededa/bin/tpmmgr writeDeviceCert
+    if ! /opt/zededa/bin/tpmmgr writeDeviceCert; then
+        echo "Failed to backup device cert in TPM NVRAM"
```

Comment:
What happens in the caller and to the system in this case?

Do you need to fallback to nor using the tpm?

Reply:
This is to backup a copy of device cert in NVRAM, in addition to /config. This is done, to avoid re-creating a new certificate if /config/device.cert.pem is deleted. This is a desired-feature, and mandatory. IOW, one need not stop using TPM just because of this failure. But a failure is logged, to inform the user about this condition.

Reply:
OK